### ISSUE TITLE
error builder and loading builder for ScalableImageWidget

### ISSUE THREAD

**Issue by HosamHasanRamadan** [zathras/jovial_svg#11]
provide `error_builder` and `loading_builder` to ScalableImageWidget and animated transition builder if possible would be nice addition but `error_builder` is a must because it is not handled by the library
<details>
<summary>stacktrace</summary>

```bash
[ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled Exception: Request to https://raw.githubusercontent.com/primer/octicons/main/icons/select-single-16.svg failed with status 404: Not Found.
#0      BaseClient._checkResponseSuccess
package:http/src/base_client.dart:103
#1      BaseClient.read
package:http/src/base_client.dart:52
<asynchronous suspension>
#2      _withClient
package:http/http.dart:164
<asynchronous suspension>
#3      ScalableImage.fromSvgHttpUrl
package:jovial_svg/src/exported.dart:356
<asynchronous suspension>
```
</details>

**Comment by zathras** [comment]
Makes sense.

**Comment by zathras** [comment]
Published in 1.0.7
